forked-daapd.conf.in

```diff
@@ -164,6 +164,15 @@ library {
 	# there is data to be read. To exclude specific pipes from watching,
 	# consider using the above _ignore options.
 #	pipe_autostart = true
+
+	# Use playback volume contained in named metadata pipes (as transmitted by
+	# shairport-sync).
+	# Enabling this option allows instant volume changes using the player's,
+	# master volume level, however the source audio written to the named pipe
+	# must then not use software volume control in addition.
+	# For shairport-sync set the option (ignore_volume_control = "yes") to
+	# disable software volume control in the pipe audio output.
+#	pipe_metadata_use_volume = false
```

Comment:
Maybe have it default true? I'm guessing most people who go to the effort of setting up a shairport metadata pipe will be interested in this.